README.md

```diff
@@ -135,6 +154,15 @@ tectonic ./main.tex
 
 * Following this project, Malcolm Ramsay has a great [blog post](https://malramsay.com/post/compiling_latex_on_travis/) documenting this use case in a very complete way.
 
+## <a name="texlive-docker">Instructions for building with TeX Live from a Docker image</a>
+
+* Install the Travis GitHub App by going to the [Marketplace](https://github.com/marketplace/travis-ci), scroll down, select Open Source (also when you want to use private repos) and select 'Install it for free', then 'Complete order and begin installation'. 
+* Now you should be in Personal settings | Applications | Travis CI | Configure and you can allow access to repositories, either select repos or all repos.
+* Copy [`3-texlive-docker/.travis.yml`](3-texlive-docker/.travis.yml) and specify which tex files you want to build in the `build-pattern` option.
+* Add all the required LaTeX packages to the `packages` option, by checking at https://www.ctan.org/pkg/some-package to see in which TeX Live package it is contained (which may be different than the LaTeX package name).
+* Commit and push, you can view your repositories at [travis-ci.com](https://travis-ci.com/).
+* For deploying to GitHub releases, see the notes [below](#deploy).
```

Comment:
Not yet supported though?

Reply:
As I said, I hope it still works. Maybe you can test it?